SI/Sem.05/Pract.05/Task03.cpp

```diff
@@ -0,0 +1,37 @@
+#include <iostream>
+
+using namespace std;
+
+void swap(int &a, int &b)
+{
+    int temp = a;
+    a = b;
+    b = temp;
+}
+
+void print(int* array, int size)
+{
+    for (int i = 0; i < size; i++)
+    {
+        cout << array[i];
+    }
+    cout << endl;
+}
+
+int main()
+{
+    int N, size = 0;
+    cin >> N;
+    
+    int binary[100];
+    while(N != 0)
```

Comment:
Revise this task.
Write a toBinary function ,which returns the number in binary.
Think of a solutions without swapping